nexus/reconfigurator/planning/src/mgs_updates/test_helpers.rs

```diff
@@ -323,125 +325,144 @@ impl TestBoards {
                 KnownArtifactKind::ControlPlane.into(),
                 ARTIFACT_HASH_CONTROL_PLANE,
                 None,
+                None,
             ),
             make_artifact(
                 "nexus",
                 KnownArtifactKind::Zone.into(),
                 ARTIFACT_HASH_NEXUS,
                 None,
+                None,
             ),
             make_artifact(
                 "host-os-phase-1",
                 ArtifactKind::HOST_PHASE_1,
                 ARTIFACT_HASH_HOST_PHASE_1,
                 None,
+                None,
             ),
             make_artifact(
                 "host-os-phase-2",
                 ArtifactKind::HOST_PHASE_2,
                 ARTIFACT_HASH_HOST_PHASE_2,
                 None,
+                None,
             ),
             make_artifact(
                 "gimlet-d",
                 KnownArtifactKind::GimletSp.into(),
                 test_artifact_for_board("gimlet-d"),
+                Some("gimlet-d"),
                 None,
             ),
             make_artifact(
                 "gimlet-e",
                 KnownArtifactKind::GimletSp.into(),
                 test_artifact_for_board("gimlet-e"),
+                Some("gimlet-e"),
                 None,
             ),
             make_artifact(
                 "sidecar-b",
                 KnownArtifactKind::SwitchSp.into(),
                 test_artifact_for_board("sidecar-b"),
+                Some("sidecar-b"),
                 None,
             ),
             make_artifact(
                 "sidecar-c",
                 KnownArtifactKind::SwitchSp.into(),
                 test_artifact_for_board("sidecar-c"),
+                Some("sidecar-c"),
                 None,
             ),
             make_artifact(
                 "psc-b",
                 KnownArtifactKind::PscSp.into(),
                 test_artifact_for_board("psc-b"),
+                Some("psc-b"),
                 None,
             ),
             make_artifact(
                 "psc-c",
                 KnownArtifactKind::PscSp.into(),
                 test_artifact_for_board("psc-c"),
+                Some("psc-c"),
                 None,
             ),
             make_artifact(
-                "oxide-rot-1",
+                "oxide-rot-1-fake-key",
```

Comment:
I'm guessing you changed this to verify that we're not accidentally depending on the target's name?

Reply:
Yep, exactly.